worker/worker.go

```diff
@@ -39,22 +39,23 @@ import (
 )
 
 type (
-	// Worker hosts workflow and activity implementations.
-	// Use worker.New(...) to create an instance.
+	// Worker is the entity that hosts and executes Workflow Tasks and Activity Tasks.
+	// Use worker.New(...) to create a new instance.
 	Worker interface {
 		Registry
 
-		// Start the worker in a non-blocking fashion.
+		// Start the Worker in a non-blocking fashion.
 		Start() error
 
-		// Run the worker in a blocking fashion. Stop the worker when interruptCh receives signal.
-		// Pass worker.InterruptCh() to stop the worker with SIGINT or SIGTERM.
-		// Pass nil to stop the worker with external Stop() call.
-		// Pass any other `<-chan interface{}` and Run will wait for signal from that channel.
-		// Returns error only if worker fails to start.
+		// Run the Worker in a blocking fashion.
+		// The Worker stops when interruptCh receives a signal.
```

Comment:
FWIW, and these docs were written way before me, I think Godoc is clearer to read if it's just capped at 80 chars a line and the sentences come one after another. Adding a newline for each sentence does not affect its output (that whitespace is trimmed).